src/Workspaces/Core/Portable/Remote/WellKnownSynchronizationKind.cs

```diff
@@ -28,7 +28,6 @@ internal enum WellKnownSynchronizationKind
     ProjectReference,
     MetadataReference,
     AnalyzerReference,
-    SourceText,
```

Comment:
removed this kind entirely.  the only thing we sujpport serializing are SerializableSourceText